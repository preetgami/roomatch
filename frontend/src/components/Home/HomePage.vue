<template>
  <!-- HTML -->

  <div
    class="mt-32 flex flex-col items-center justify-center text-center sm:mt-40"
  >
    <TopHomePage />
    <br /><br />
    <SignupScroller />
    <br /><br /><br />
    <div class="grid grid-cols-2 gap-4">
      <OurServices />
    </div>
    <br /><br /><br /><br /><br />
  </div>
</template>
<script>
import OurServices from "./OurServices.vue";
import TopHomePage from "./TopHomePage.vue";
import SignupScroller from "./SignUpScroller.vue";

export default {
  name: "HomePage",
  components: {
    OurServices,
    SignupScroller,
    TopHomePage,
  },
};
</script>
