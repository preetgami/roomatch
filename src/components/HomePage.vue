<template>
  <h1>home stuff goes here</h1>
</template>
<script>
export default {};
</script>
