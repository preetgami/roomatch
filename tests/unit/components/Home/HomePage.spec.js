import { shallowMount } from '@vue/test-utils';
import HomePage from '@/components/Home/HomePage.vue';

describe('Homepage', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallowMount(HomePage);
    });

    it('exists', () => {
        expect(HomePage).toBeDefined();
    });

    it('name matches', () => {
        expect(HomePage.name).toBe('HomesPage');
    });

    it('snapshot matches', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });
});
