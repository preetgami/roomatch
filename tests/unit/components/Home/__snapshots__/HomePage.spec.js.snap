// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage snapshot matches 1`] = `
<!-- HTML -->
<div class="mt-32 flex flex-col items-center justify-center text-center sm:mt-40">
  <top-home-page-stub></top-home-page-stub><br><br>
  <signup-scroller-stub></signup-scroller-stub><br><br><br>
  <div class="grid grid-cols-2 gap-4">
    <our-services-stub></our-services-stub>
  </div><br><br><br><br><br>
</div>
`;
